SoftLayer/CLI/virt/host_list.py

```diff
@@ -0,0 +1,108 @@
+"""List dedicated hosts."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI.command import SLCommand as SLCommand
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer import utils
+
+
+@click.command(cls=SLCommand, short_help="List virtual servers.")
+@click.option('--name', '-n', help='Filter by name of the dedicated host')
```

Comment:
In this part just a little suggestion, maybe will be better sort the params in alphabetical order, so that it is the closest thing to the original command

Reply:
Done